app-network/ucx/autobuild/defines

```diff
@@ -0,0 +1,24 @@
+PKGNAME=ucx
+PKGSEC=science
+PKGDEP="gcc glibc libtool pkg-config rdma-core fuse-3"
+BUILDDEP="autoconf automake m4"
+BUILDDEP__AMD64="${BUILDDEP} cuda valgrind"
+BUILDDEP__ARM64="${BUILDDEP} cuda valgrind"
+PKGDES="Unified Communication Library"
+PKGSUG="cuda"
+
+NOSTATIC=0
+AUTOTOOLS_AFTER="--libdir=/usr/lib/ucx \
```

Comment:
Why `/usr/lib/ucx`? I think using the default `/usr/lib` is okay, and we can remote the extra ld.conf.

Reply:
I referred to OpenMPI's packaging scripts(app-scientific/openmpi) and I once thought this was the convention of AOSC.

Reply:
OpenMPI is different: you may have multiple MPI implementations, but ucx is unique.